harborclient/harborclient.py

```diff
@@ -1,13 +1,16 @@
 #!/usr/bin/env python
 
-import requests
-import simplejson
+# for inner
+import json
 import logging
+# for thirdparty
```

Comment:
Sorry about my statement. I'm confused about the comment and thanks for your explaination.